### PR TITLE
Satvik6220 launcher test

### PR DESCRIPTION
**Pull request by Prime-235711**
This is code to run the launcher as well as read out the launcher rpm

### CODE REVIEW

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java [hunk 1]
```diff
@@ -55,4 +64,48 @@ public void driveMecanum(double driveAngle, double drivePower, double w)
             motorBackRight.setPower(motorBRPower);
         }
     }
+
+    //Sets the launch motor to a given power
+    public void driveLauncher(double power){
+        motorLauncher.setPower(power);
+    }
+
+    //This method returns the speed of a given motor after a delay od delayInMillis
```
Comment (malavika1) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: of

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java [hunk 2]
```diff
@@ -55,4 +64,48 @@ public void driveMecanum(double driveAngle, double drivePower, double w)
...
+
+        endTime = System.currentTimeMillis();
+        endPosition = motor.getCurrentPosition();
+
+        //Calculates the ▲Position and the ▲Time
+        positionChange = endPosition - startPosition;
+        timeChange = endTime - startTime;
+
+        //Converts the ▲Time from milliseconds to minutes then finds encoder ticks per minute
+        double timeChangeInMin = timeChange / 60000;
```
Comment (malavika1) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: This 60000 should be a constant

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java (hunk 1 above)
Comment (Prime-235711) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: what?

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java (hunk 2 above)
Comment (Prime-235711) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: ok

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java (hunk 1 above)
Comment (malavika1) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: You typed "od" instead of "of"

Comment (Prime-235711) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: oh, ill fix it

Comment (Prime-235711) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: ok done

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java (hunk 2 above)
Comment (Prime-235711) [reply Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterOpMode.java]: ok done

**Review by malavika1 — APPROVED** [review]

**Pull request merged**